README.md

```diff
@@ -318,6 +318,14 @@ var output = translate.prettyPrintLegacyAssemblyJSON(assemblyJSON, sourceCode)
 
 ## Browser Usage
 
+**NOTE**: We currently compile the wasm module in `soljson.js` synchronously (see [here](https://github.com/ethereum/solc-js/issues/581) for more information), which may not work in Chrome-based
+browsers since it limits the size of modules that can be compiled/fetched synchronously to avoid blocking the main thread.
+Thus, if you get an error like: `WebAssembly.Compile is disallowed on the main thread, if the buffer size is larger than 4KB`,
+when trying to use the method described [below](#directly-loading-the-module-does-not-work-on-chrome-based-browsers),
+please consider trying to use `solc` through a [web worker](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers) as described [here](#loading-solc-with-web-workers) instead.
+
+### Directly loading the module (does not work on chrome-based browsers)
```

Comment:
Should we just remove this section entirely and suggest to *always* use web workers generally? People will rarely want to do anything that only works on a fraction of browsers anyways, right?

Fine with keeping it as well, though.

Reply:
Right. Indeed, maybe it is better to just recommend to always use web workers as you said. This can also avoid some confusions. I will update the readme.